build_defs.bzl

```
@@ -56,6 +57,9 @@ def generate_prime_field(
     else:
         arm64_s_out = name + "_raw_arm64.s"
 
+    if no_adx != None or no_adx == True:
```

```suggestion
    if no_adx:
```
How about like this??

---

changed as suggested